src/app/components/progress-indicator/index.html

```diff
@@ -103,31 +103,31 @@
     <p>
       The progress indicator nav button component allows users to navigate the steps in modal wizards. We recommend against using it in waterfall pages and passive progress indicators.
     </p>
+    <sky-demo-page-property
+      propertyName="progressIndicator"
+    >
+      Specifies the progress indicator component to associate with the nav button. This property accepts an array or Observable array of <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="buttonText"
       isOptional="true"
       defaultValue="Next"
     >
-      Specifies the label to display on the nav button.
+      Specifies the label to display on the nav button. The property accepts <stache-code>string</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="buttonType"
       isOptional="true"
       defaultValue="next"
     >
-      Specifies the type of nav button to include. Options include <stache-code>finish</stache-code>, <stache-code>next</stache-code>, and <stache-code>previous</stache-code>.
+      Specifies the type of nav button to include. Options include <stache-code>finish</stache-code>, <stache-code>next</stache-code>, and <stache-code>previous</stache-code>. This property accepts <stache-code>string</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="disabled"
       isOptional="true"
       defaultValue="false"
     >
-      Indicates whether to disable the nav button.
-    </sky-demo-page-property>
-    <sky-demo-page-property
-      propertyName="progressIndicator"
-    >
-      Specifies the progress indicator component to associate with the nav button.
+      Indicates whether to disable the nav button. This property accepts <stache-code>boolean</stache-code> values.
     </sky-demo-page-property>
   </sky-demo-page-properties>
 
```

Comment:
We don't have a section for the progress indicator title component (https://github.com/blackbaud/skyux-progress-indicator/blob/master/src/app/public/modules/progress-indicator/progress-indicator-title/progress-indicator-title.component.ts) because it doesn't have public inputs or outputs. However, I suspect we should have something about this component because we use it in the demo. Maybe we should have a child components section (like in the toolbar component docs and one or two others) where we list all of the related components. It's probably worth asking one of the devs about this.

Reply:
I _think_ I handled this the way we chatted about but definitely going to lean on the extra set of eyes for this one.

Reply:
I'm making a few tweaks. For the title component, you had "component" in the code snippet, but we didn't need that because it's not part of the selector. And I swapped the order to use alphabetic order. I also added entries for the nav button and reset button components. And I moved this section to the beginning instead of the end. I think it makes more sense at the start than after we've already listed all the properties for the child components.